examples/out-of-tree/credential-provider-config.yaml

```diff
@@ -9,5 +9,7 @@ providers:
   - "*.azurecr.cn"
   - "*.azurecr.de"
   - "*.azurecr.us"
+  - "mcr.microsoft.com"
```

Comment:
is it required?

Reply:
It is required when testing on my local setup.